src/main/java/seedu/address/logic/TextToSpeech.java

```diff
@@ -0,0 +1,79 @@
+package seedu.address.logic;
+
+import java.io.File;
+import java.io.FileOutputStream;
+
+import com.voicerss.tts.AudioCodec;
+import com.voicerss.tts.AudioFormat;
+import com.voicerss.tts.Languages;
+import com.voicerss.tts.SpeechDataEvent;
+import com.voicerss.tts.SpeechDataEventListener;
+import com.voicerss.tts.SpeechErrorEvent;
+import com.voicerss.tts.SpeechErrorEventListener;
+import com.voicerss.tts.VoiceParameters;
+import com.voicerss.tts.VoiceProvider;
+
+import javafx.scene.media.Media;
+import javafx.scene.media.MediaPlayer;
+
+/**
+ * Text To Speech
+ */
+public class TextToSpeech {
+
+    private static MediaPlayer mediaPlayer;
+
+    private String word;
+
+    public TextToSpeech(String word) {
+        this.word = word;
+        execute();
+    }
+
+    /**
+     * Execute the Text to Speech Function
+     * and returns void.
+     */
+    public void execute() {
+        VoiceProvider tts = new VoiceProvider("c580b327590b4cb881deee28bf85a543");
```

Comment:
Is it possible for VoiceProvider to be a static instance? This way, we don't instantiate a new one every time a new TextToSpeech is created (which is every time a command is run).